resolwe_bio/processes/clustering/pca.py

```diff
@@ -0,0 +1,334 @@
+"""Beta principal component analysis process."""
+
+import json
+
+import numpy as np
+import pandas as pd
+from sklearn.decomposition import PCA as plotPCA
```

Comment:
If you decide to rename the class to `PrincipalComponentAnalysis` (see below), you can import this using the native name `PCA`.